examples/voice_send.cr

```diff
@@ -50,7 +50,7 @@ client.on_message_create do |payload|
     # The DCAParser class handles parsing of the DCA file. It doesn't do any
     # sending of audio data to Discord itself – that has to be done by
     # VoiceClient.
-    parser = Discord::DCAParser.new(file)
+    parser = Discord::DCAParser.new(file, strict_metadata: false)
```

Comment:
Because people might not care about it, and thus not bother to fill the data fields out as required.